plugins/rss/help.md

```diff
@@ -61,133 +63,79 @@ Example input:
-    "summary": "This is a good article on the complicated story of hacker Marcus Hutchins....",
-    "summary_detail": {
-      "type": "text/plain",
-      "language": null,
-      "base": "https://www.schneier.com/blog/atom.xml",
-      "value": "This is a good article on the complicated story of hacker Marcus Hutchins...."
-    },
-    "authors": [
+    "id": "http://example.org/guid/1",
+    "link": "http://example.org/item/1",
+    "links": [
+      {
+        "href": "http://example.org/item/1",
+        "rel": "alternate",
+        "type": "text/html"
+      },
       {
-        "name": "Bruce Schneier"
+        "rel": "enclosure",
+        "url": ""
       }
     ],
-    "author_detail": {
-      "name": "Bruce Schneier"
+    "published": "Thu, 05 Sep 2002 0:00:01 GMT",
+    "summary": "Watch out for\n<span>\nnasty tricks</span>",
+    "summary_detail": {
+      "base": "",
+      "language": null,
+      "type": "text/html",
+      "value": "Watch out for\n<span>\nnasty tricks</span>"
     },
-    "author": "Bruce Schneier",
     "tags": [
       {
-        "term": "bitcoin",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "bitcoin"
-      },
-      {
-        "term": "cybersecurity",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "cybersecurity"
-      },
-      {
-        "term": "fraud",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "fraud"
-      },
-      {
-        "term": "hacking",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "hacking"
-      },
-      {
-        "term": "killswitch",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "kill switch"
-      },
-      {
-        "term": "privacy",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "privacy"
-      },
-      {
-        "term": "ransomware",
-        "scheme": "http://www.sixapart.com/ns/types#tag",
-        "label": "ransomware"
+        "label": null,
+        "scheme": null,
+        "term": "Miscellaneous"
       }
     ],
-    "content": [
-      {
-        "type": "text/html",
-        "language": "en-us",
-        "base": "https://www.schneier.com/blog/",
-        "value": "<p><a href=\"https://nymag.com/selectall/2018/03/marcus-hutchins-hacker.html\">This</a> is a good article on the complicated story of hacker Marcus Hutchins.</p>"
-      }
-    ]
+    "title": "First item title",
+    "title_detail": {
+      "base": "",
+      "language": null,
+      "type": "text/plain",
+      "value": "First item title"
+    }
   }
 }
-
 ```
+### Tasks
+  
+*This plugin does not contain any tasks.*
 
-### Custom Output Types
-
-_This plugin does not contain any custom output types._
+### Custom Types
+  
+*This plugin does not contain any custom output types.*
 
 ## Troubleshooting
-
-Between workflow runs, new items will *not* be reported on.
+  
+*There is no troubleshooting for this plugin.*
```

Comment:
Do we still need this? The tooling doesn't support troubleshooting generation atm so it usually clears what already exists
```suggestion
Between workflow runs, new items will *not* be reported on.
```

Reply:
You are right. Didn't notice so. I'll update it.